weasyl/test/test_note.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import absolute_import, unicode_literals
+
+import pytest
+
+from libweasyl import staff
+from weasyl import define as d
+from weasyl import note
+from weasyl import shout
+from weasyl.error import WeasylError
+from weasyl.test import db_utils
+from weasyl.test.utils import Bag
+
+
+def _make_test_note(recipient="user2", title="ThisIsATestNote", content="ThisIsTheTestContent", mod_copy='', staff_note=''):
+    return Bag(
+        recipient=recipient,
+        title=title,
+        content=content,
+        mod_copy=mod_copy,
+        staff_note=staff_note,
+    )
+
+
+def _set_user_note_config(userid=None, set_note_config=None, app=None):
+    """
+    Set the note configuration for a user
+    :param userid:
+    :param set_note_config: 'z' is friends only notes, 'y' is staff only, 'anyone' is all users
+    :return: Nothing.
+    """
+    cookie = db_utils.create_session(userid)
+    payload = {
+        'rating': 10,
+        'sfwrating': 10,
+        'timezone': 'America/Dawson',
+        'notes': set_note_config if set_note_config is not None else "anyone",
+    }
+    app.post(url='/control/editpreferences', params=payload, headers={'Cookie': cookie})
+
+
+@pytest.fixture(autouse=True)
```

Comment:
It doesn’t look like this should be `autouse`d – everything that needs it has a `create_users` parameter.